DemoApp/ChatPresenter.swift

```diff
@@ -252,26 +252,35 @@ class DemoChatChannelListRouter: ChatChannelListRouter {
             }),
             (
                 channelController.channel?.isHidden == false ?
-                    .init(title: "Hide channel", style: .default, handler: { _ in
+                    .init(title: "Hide channel", style: .default, handler: { [unowned self] _ in
                         channelController.hideChannel { error in
                             if let error = error {
                                 self.rootViewController.presentAlert(title: "Couldn't hide channel \(cid)", message: "\(error)")
                             }
                         }
                     }) :
-                    .init(title: "Show channel", style: .default, handler: { _ in
+                    .init(title: "Show channel", style: .default, handler: { [unowned self] _ in
                         channelController.showChannel() { error in
                             if let error = error {
-                                self.rootViewController.presentAlert(title: "Couldn't unhide channel \(cid)", message: "\(error)")
+                                self.rootViewController.presentAlert(
+                                    title: "Couldn't unhide channel \(cid)",
+                                    message: "\(error)"
+                                )
                             }
                         }
                     })
             ),
-            .init(title: "Show Channel Info", style: .default, handler: { _ in
+            .init(title: "Show Channel Info", style: .default, handler: { [unowned self] _ in
                 self.rootViewController.presentAlert(
                     title: "Channel Info",
                     message: channelController.channel.debugDescription
                 )
+            }),
+            .init(title: "Truncate channel w/o message", style: .default, handler: { _ in
```

Comment:
Like other actions, we should show an alert if the action fails with an error

Reply:
On it!

Reply:
Done!